web/modules/custom/rdf_entity/rdf_entity.routing.yml

```diff
@@ -55,3 +55,51 @@ rdf_entity.rdf_entity_settings:
     _title: 'RDF Settings'
   requirements:
     _permission: 'administer rdf entity'
+
```

Comment:
This `rdf_entity.rdf_entity_settings` route seems to be obsolete. It leads to an empty page.